lib/jsonwp-proxy/proxy.js

```diff
@@ -41,7 +41,7 @@ class JWProxy {
       keepAlive: opts.keepAlive ?? true,
       maxSockets: 10,
       maxFreeSockets: 5,
-      timeout: 60000,
+      timeout: this.timeout,
```

Comment:
I assume this timeout has no effect, because it gets overridden in reqOpts. I would say it is safe to remove it completely to avoid confusion

Reply:
Okay, thanks. I read the documentation, but haven't decided which was (keep or remove it) better